assets/apps_mac.json

```diff
@@ -0,0 +1,6 @@
+{
+	"env":{
+		"PATH":"$(PATH):$(HOME)/.local/bin"
+	},
+	"apps":[ ]
```

Comment:
- Is it possible to add a Steam BigPicture placeholder, like we do for Windows and Linux?
- Does the Desktop show up by default, like on Windows? Or do we need to add it like on Linux?

Reply:
I have to research that. Currently the implementation only supports streaming the entire desktop of the primary screen. So that answers also the second question, the desktop will show up by default.